.mega-linter.yml

```diff
@@ -1,18 +1,18 @@
 ENABLE:
-  - BASH
-  - DOCKERFILE
-  - EDITORCONFIG
-  - JSON
-  - MARKDOWN
+  - PYTHON
```

Comment:
Why are you removing these linters? They are used for the containerdev environment.